websites/models.py

```diff
@@ -342,6 +344,12 @@ class WebsiteStarter(TimestampedModel):
         null=False,
         help_text="Human-friendly name of the starter project.",
     )
+    status = models.CharField(
+        max_length=30,
+        choices=WEBSITE_STARTER_STATUS_CHOICES,
```

Comment:
I haven't followed the conventional approach for this choice field (i.e: `zip(choices_list, choices_list)`) as I think keys should have a different list than the values. As values might change, or have spaces, or have upper/lower case stuff etc which keys can not have, so a class named `WebsiteStarterStatus` has been created to store the keys only, and choices util from `django-model-utils` is being used to construct the choices structure, and this choices obj/structure is used in model. 

To me, this approach looks a bit clean and smooth but I'm open to suggestions. 
And if this approach is fine, then I think, for future choices fields, we can follow this approach and for previous choices fields, maybe we can change/improve them with time. (if required)